homeassistant/components/netatmo/sensor.py

```diff
@@ -581,23 +447,31 @@ def __init__(
         """Initialize the sensor."""
         super().__init__(netatmo_device.data_handler)
         self.entity_description = NetatmoSensorEntityDescription(
-            key="battery_percent",
+            key="battery",
```

Comment:
Breaking change? Affects unique id

Reply:
I only makes a couple of them but all keys which are changed should result in a breaking change. Why don't you revert it or convert the old unique ids into new ones? There is no reason for a breaking change due to changed unique ids here

Reply:
Thanks, I reverted most breaking changes. Two get dropped though but those were only diagnostics.

Reply:
👍🏻